scripts/inspect.sh

```diff
@@ -357,6 +362,16 @@ function suggest_fixes {
 Refer to https://microk8s.io/docs for instructions on air-gap deployments.\n\n
 "
   fi
+
+  if ! is_strict
+  then
+    AIO_MAX_NR=$(sysctl -n fs.aio-max-nr)
+    AIO_NR=$(sysctl -n fs.aio-nr)
+    if [ "$AIO_NR" -ge "$AIO_MAX_NR" ]; then
+      printf -- "\033[0;33mWARNING: \033[0m Available asyncio requests are exhausted. This might lead to dqlite being unresponsive. \n"
+      printf -- "\t  Increase the limit with: 'sudo sysctl fs.aio-max-nr=3145728' and reboot k8s-dqlite service. \n"
```

Comment:
"reboot k8s-dqlite service" might be confusing, how about this instead?
```suggestion
      printf -- "\t  Increase the limit and restart the k8s-dqlite service with: \n"
      printf -- "\t  \t sudo sysctl fs.aio.max-nr=3145728\n"
      printf -- "\t  \t sudo snap restart microk8s.daemon-k8s-dqlite\n"
```

Reply:
👍 Definitely sounds better, thanks!